apps/docs/content/components/Grid/index.mdx

```diff
@@ -127,175 +127,49 @@ We recommend using either the grid or stack component, depending on how the chil
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/022fc146-6286-4074-abc5-7341743da82e"
-    role="presentation"
-    width="456"
-  />
-</Container>
-
-<Container
-  as="figure"
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  m={0}
-  mb={3}
-  sx={{gap: 3}}
->
-  <Container as="figcaption" mt="1">
-    <h4 style={{marginBottom: 0}}>List of actions</h4>
-    <Text as="p" mt="0">
-      For a group of action or buttons, where items are displayed next to each
-      other, and they are distributed horizontally or vertically, depending on
-      the content length and the screen size.
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/0b1025c7-1c07-4ff3-adab-5d685d78ea99"
-    role="presentation"
-    width="456"
-  />
-</Container>
-
-<Container
-  as="figure"
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  m={0}
-  mb={3}
-  sx={{gap: 3}}
->
-  <Container as="figcaption" mt="1">
-    <h4 style={{marginBottom: 0}}>Form elements</h4>
-    <Text as="p" mt="0">
-      For combinations of input fields with a button next to it.
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/8f83eae5-995b-426a-bf54-7407579cdb8c"
-    role="presentation"
-    width="456"
-  />
-</Container>
-
-<Container
-  as="figure"
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  m={0}
-  mb={3}
-  sx={{gap: 3}}
->
-  <Container as="figcaption" mt="1">
-    <h4 style={{marginBottom: 0}}>Profile cards</h4>
-    <Text as="p" mt="0">
-      For combinations of an avatar and descriptive content, where the
-      information takes the remaining space from its parent element. This could
-      be applied to any combinations of an icon and text too.
-    </Text>
-  </Container>
-  <img
-    src="https://github.com/primer/brand/assets/912236/c53f2ee7-93f1-416f-8fd5-bd965fd37f86"
-    role="presentation"
-    width="456"
-  />
-</Container>
+#### Navigation
+
+For the page navigation or a list of links. E.g. Having a logo on the left side and the navigation on the right.
+
+![](https://github.com/primer/brand/assets/912236/022fc146-6286-4074-abc5-7341743da82e)
+
+#### List of actions
+
+For a group of action or buttons, where items are displayed next to each other, and they are distributed horizontally or vertically, depending on the content length and the screen size.
+
+![](https://github.com/primer/brand/assets/912236/10ae15dd-f8f9-4174-9c89-2e9355120204)
+
+#### Form elements
+
+For combinations of input fields with a button next to it.
+
+![](https://github.com/primer/brand/assets/912236/78a81f0e-55d5-4db5-8a8b-3184394fac51)
+
+#### Profile cards
```

Comment:
These don't visually appear like headings, which is presumably a theme limitation. Can we make this a h3 instead and bump up each preceding heading?

![Screenshot 2023-10-03 at 11 44 05](https://github.com/primer/brand/assets/13340707/38341c9b-6192-424f-ab88-0c228000509d)

Reply:
I think I rather adjust the theme in Doctocat than bumping each upper level heading, as this section deserves 3 levels at least 👍